examples/Billing/AddBillingSetup.cs

```diff
@@ -0,0 +1,205 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Ads.GoogleAds.Lib;
+using Google.Ads.GoogleAds.V3.Errors;
+using Google.Ads.GoogleAds.V3.Resources;
+using Google.Ads.GoogleAds.V3.Services;
+using Google.Api.Gax;
+using System;
+using System.Linq;
+using static Google.Ads.GoogleAds.V3.Enums.TimeTypeEnum.Types;
+
+namespace Google.Ads.GoogleAds.Examples.V3
+{
+    /// <summary>
+    /// This example creates a billing setup for a customer. A billing setup is a link between
+    /// payment account and customer. In the case of consolidated billing, a payments account is
+    /// linked to the manager account and is linked to a customer account via a billing setup.
+    /// </summary>
+    public class AddBillingSetup : ExampleBase
+    {
+        /// <summary>
+        /// Main method, to run this code example as a standalone application.
+        /// </summary>
+        /// <param name="args">The command line arguments.</param>
+        public static void Main(string[] args)
+        {
+            AddBillingSetup codeExample = new AddBillingSetup();
+
+            Console.WriteLine(codeExample.Description);
+
+            // The Google Ads customer ID for which the call is made.
+            long customerId = long.Parse("INSERT_CUSTOMER_ID_HERE");
+
+            // Payments setup identifiers. Provide an existing payments account ID to link to the
+            // new billing setup. Must be formatted as "1234-5678-9012-3456".
+            string paymentsAccountId = "INSERT_PAYMENTS_ACCOUNT_HERE";
+            // Alternatively, provide a payments profile ID, which will be linked to a new payments
```

Comment:
Either the payments account ID or the payments profile ID has to be provided for the example to run successfully. What do you think of adding an extra explanation somewhere stating which of those two parameters will be used if both are specified?

Reply:
Good idea; expanded the summary comment to the Run and Main methods to this effect.